### DISCUSSION TITLE
Rtsp camera plugin problem!

### DISCUSSION THREAD

**Discussion by maxweb80** [koush/scrypted#383] [Q&A]
In RTSP camera plugin settings, I no longer have the card "integration and Extensions" ! why???

**Comment by iblis82** [comment]
Same issue here. 
Running Scrypted version 0.2.9 on a Mac Mini (intel).

![7D31BCA8-C586-4BE7-BFE8-40E91B91F9A7](https://user-images.githubusercontent.com/14071775/193135357-727f2cf4-2e5b-4dd7-909a-8adb029e26a9.jpeg)

**Reply by koush** [reply]
Taking a look
